lib/services/firebase_service.dart

```diff
@@ -425,4 +426,33 @@ class FirebaseService extends IFirebaseService {
           '$onError Failed on update $data to $collection $docId $subCollection $subCollectionDoc');
     });
   }
+
+  Future<List<QueryDocumentSnapshot>> getAppointmentList(
+      {@required DateTime currentDate}) async {
+    final DateFormat formatter = DateFormat('yyyy-MM-dd');
+    String formattedStart = formatter.format(currentDate);
+    var startWithOutTime = DateTime.parse(formattedStart);
+    var start = _calculationService.formatDate(date: startWithOutTime);
+    print(
+        'selected Date is ${_calculationService.formatDate(date: currentDate)}');
+    DateTime end = start.add(Duration(days: 1));
+    print('start == $start');
+    print('END = = $end');
+    print('start to UDfsifods == ${start.isBefore(end)}');
+    var data = await _firestore
+        .collection('Appointments')
+        .where('date', isGreaterThan: start)
+        .where('date', isLessThan: end)
+        .get()
+        .then((value) {
+      print(value.docs.first.data());
+      return value.docs;
+    }).catchError((onError) {
+      print('$onError WTF BRO');
+      return null;
+    });
+    print(data.length);
+    print('endFunction');
+    return data;
+  }
```

Comment:
currentDate มาจากวันที่ขึ้นโชว์ของหน้าตาราง ถ้ากดเลื่อนวันมันก็จะต้องขึ้นเป็นของวันถัดไป ประเด็นคือถ้าเป็นเสาร์ที่ 23 มันโชว์มาได้ แต่ถ้ากดเป็นวันพรุ่งนี้มันไม่มา 
![image](https://user-images.githubusercontent.com/57885091/105542353-cc730900-5d2b-11eb-80e0-4f401d721525.png)


ในdb ก็มีนะ ... 
![image](https://user-images.githubusercontent.com/57885091/105542572-093f0000-5d2c-11eb-9284-089cdcc6bbd1.png)

Reply:
![image](https://user-images.githubusercontent.com/57885091/105543366-4acfab00-5d2c-11eb-9ff7-d83fa4172ba7.png)

Reply:
Fix in  [https://github.com/LaCare-KMUTT/AbdoCare-Web/pull/33/commits/02238870f0e68570bbd29bbd6479172a9ddda23a](url)